nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -73,9 +73,38 @@ When NiFi first starts up, the following files and directories are created:
 
 See the <<system_properties>> section of this guide for more information about configuring NiFi repositories and configuration files.
 
+== Port Configuration
+
+=== NiFi
+The following table lists the default ports used by NiFi and the corresponding property in the _nifi.properties_ file.
+
+[options="header,footer"]
+|==================================================================================================================================================
+| Function                        |  Property                             | Default Value
+|HTTP Port                        | `nifi.web.http.port`                  | `8080`
+|HTTPS Port*                      | `nifi.web.https.port`                 | `9443`
+|Remote Input Socket Port*        | `nifi.remote.input.socket.port`       | `10443`
+|Cluster Node Protocol Port*      | `nifi.cluster.node.protocol.port`     | `11443`
+|Cluster Node Load Balancing Port | `nifi.cluster.node.load.balance.port` | `6342`
+|Web HTTP Forwarding Port         | `nifi.web.http.port.forwarding`       | blank
```

Comment:
Can we represent no default value here other than "blank"?

Reply:
I was thinking about that also, but figured "blank" would be OK since we reference this throughout the Admin Guide  (e.g. "The default value is blank").

Reply:
As discussed in other comment threads, changing blank to _none_.